src/index.tsx

```diff
@@ -38,7 +38,9 @@ import '../src/assets/css/scrollStyles.css';
 const theme = createTheme({
   palette: {
     primary: {
-      main: '#31bb6b',
+      main: getComputedStyle(document.documentElement)
+        .getPropertyValue('--blue-primary')
```

Comment:
Use a variable name not the hex/rgba code (Use 8 digit HTML codes for transparency if necessary). Ensure that the variable name you choose describes the function of the color and not the color itself. This will help with future attempts to having Dark Mode.  
- For example `--search-button-bg` is much better than `--light-blue`  

Use values from `app.module.css`